### PR TITLE
TELCODOCS-891 creating my own PR for dev created PR 49350

### PR DESCRIPTION
**Pull request by kquinn1204**
GH Issue: 49350: performance creator minor updates

Version(s):
  * PR applies to all versions after a specific version: 4.11 and > and main
Issue:
https://github.com/openshift/openshift-docs/pull/49350 raised by dev 

Link to docs preview:http://file.emea.redhat.com/kquinn/TELCODOCS-891/scalability_and_performance/cnf-create-performance-profiles.html#running-the-performance-profile-profile-cluster-using-podman_cnf-create-performance-profiles

### CODE REVIEW

**Comment by kquinn1204** [comment]
Hi @yuvalk @fromanirh can you please review this PR which includes changes in https://github.com/openshift/openshift-docs/pull/49350 and some substitution updates so {product-version} will reflect 4.11 for 4.11 and 4.12 for 4.12 and so on.

**Comment by kquinn1204** [comment]
@mrniranjan sorry to ping you but AFAIK you helped on original PR, can you review this and give an LGTM if you are happy with content.

File: modules/cnf-gathering-data-about-cluster-using-must-gather.adoc [hunk 1]
```diff
@@ -32,9 +32,9 @@ $ oc adm must-gather --image=<PAO_must_gather_image> --dest-dir=<dir>
 +
 .Example
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather --image=registry.redhat.io/openshift4/performance-addon-operator-must-gather-rhel8:v4.11 --dest-dir=must-gather
+$ oc adm must-gather --image=registry.redhat.io/openshift4/performance-addon-operator-must-gather-rhel8:v{product-version} --dest-dir=must-gather
```
Comment (yuvalk) [inline modules/cnf-gathering-data-about-cluster-using-must-gather.adoc:37]: dest-dir=<path_to-must-gather>
here too?

Comment (kquinn1204) [reply modules/cnf-gathering-data-about-cluster-using-must-gather.adoc]: Hi @yuvalk thanks for feedback I picked this up. Hopefully good for LGTM

**Comment by yuvalk** [comment]
this already have the lgtm flag
but anyhow, this also lgtm
from my perspective 
ie- covers everything from my pr and even better! :-)

**Comment by yuvalk** [comment]
/close
this is covered by https://github.com/openshift/openshift-docs/pull/49527

**Comment by yuvalk** [comment]
/reopen
closed by mistake

File: modules/cnf-running-the-performance-creator-profile.adoc [hunk 2]
```diff
@@ -47,14 +47,14 @@ Password: ************
...
+[source,terminal,subs="attributes+"]
 ----
-$ podman run --entrypoint performance-profile-creator registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v4.11 -h
+$ podman run --rm --entrypoint performance-profile-creator registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v{product-version} -h
 ----
 +
 .Example output
 +
-[source,terminal]
+[source,terminal, ]
```
Comment (opayne1) [inline modules/cnf-running-the-performance-creator-profile.adoc:57]: Is there supposed to be something after the comma here?

**Review by opayne1 — COMMENTED** [review]
Just one question, other than that lgtm!

File: modules/cnf-running-the-performance-creator-profile.adoc (hunk 2 above)
Comment (kquinn1204) [reply modules/cnf-running-the-performance-creator-profile.adoc:57]: hmm.. let me look

**Pull request merged**